recipes/antlr-cpp-runtime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "antlr-cpp-runtime" %}
+{% set version = "4.12.0" %}
+{% set sha256 = "642d59854ddc0cebb5b23b2233ad0a8723eef20e66ef78b5b898d0a67556893b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.antlr.org/download/antlr4-cpp-runtime-{{ version }}-source.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
what's level of effort to support windows vs e.g. `nmake`? most `cmake` stuff seems to work _fairly_ well...

Reply:
pretty low, for most of my C++ libs I tend to get Mac+linux in first, then iterate on the feedstock vs iterating on staged-recipes.